jujugui/static/gui/src/app/components/deployment/add-credentials/add-credentials.js

```diff
@@ -125,12 +135,22 @@ YUI.add('deployment-add-credentials', function() {
                   security_credential
                 </a>
               </p>
+              <label className="deployment-panel__label"
+                htmlFor="access-key">
+                Access key
+              </label>
               <input className="deployment-panel__input"
-                placeholder="Access-key"
+                id="access-key"
+                placeholder="xxxx xxxx xxxx xxxx"
```

Comment:
I figure we should probably use a pattern which matches the access and secret keys for aws which is just a long contiguous string.